WordPress/pattern-directory
# Tooling: Create necessary content (Pages, Block Categories) on install.

It's hard without looking through code to figure out that the environment needs certain pages and certain categories to be fully functional.

This PR adds some `wp-cli` commands to install needed pages and block categories. Since these pages contain no content, I chose to use the command line interface as opposed to importing a DB export.


### How to test the changes in this Pull Request:

1. Start with a clean environment
2. Install environment as explained in Readme.md
3. Open `localhost:8888`, expect to see pattern categories
4. Click through navigational links and expect to see pages, not 404s.  (New Pattern, My Favorites, My Patterns)
